pkgs/by-name/vo/volk/package.nix

```diff
@@ -0,0 +1,46 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  fetchpatch,
+  cmake,
+  python3,
+  enableModTool ? true,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "volk";
+  version = "3.1.2";
+
+  src = fetchFromGitHub {
+    owner = "gnuradio";
+    repo = "volk";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-R1FT5sbl0fAAl6YIX5aD5CiQ/AjZkCSDPPQPiuy4WBY=";
+    fetchSubmodules = true;
+  };
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/gnuradio/volk/commit/e46771a739658b5483c25ee1203587bf07468c4d.patch";
```

Comment:
lost a comment here. is this patch still needed?

Reply:
> lost a comment here. is this patch still needed?

I forgot to comment, this patch is a different patch: https://github.com/gnuradio/volk/pull/766 .

Reply:
Now https://github.com/gnuradio/volk/pull/766 is mentioned in a comment.